src/Hal/Report/Html/template/js/graph-maintainability.js

```diff
@@ -37,31 +37,48 @@ function chartMaintainability() {
             return d.r;
         })
         .style("fill", function (d) {
-            if (d.mi > 85) {
-                return '#8BC34A';
-            } else if (d.mi > 69) {
-                return '#FFC107';
+            if (true === withoutComment) {
+                if (d.mIwoC > 65) {
+                    return '#8BC34A';
+                } else if (d.mIwoC > 53) {
+                    return '#FFC107';
+                } else {
+                    return '#F44336';
+                }
             } else {
-                return '#F44336';
+                if (d.mi > 85) {
+                    return '#8BC34A';
+                } else if (d.mi > 69) {
+                    return '#FFC107';
+                } else {
+                    return '#F44336';
+                }
             }
         })
-        .attr("transform", function (d) {
-            return "translate(" + d.x + "," + d.y + ")";
-        })
         .on('mouseover', function (d) {
-            var text = '<strong>' + d.name + '</strong>'
-                + "<br />Cyclomatic Complexity : " + d.ccn
-                + "<br />Maintainability Index: " + d.mi;
+            if (true === withoutComment) {
+                var text = '<strong>' + d.name + '</strong>'
+                    + "<br />Cyclomatic Complexity : " + d.ccn
+                    + "<br />Maintainability Index (w/o comments): " + d.mIwoC;
+            } else {
+                var text = '<strong>' + d.name + '</strong>'
+                    + "<br />Cyclomatic Complexity : " + d.ccn
+                    + "<br />Maintainability Index: " + d.mi;
+            }
             d3.select('.tooltip').html(text);
-            d3.select(".tooltip").style("opacity", 1);
+            d3.select(".tooltip")
+                .style("opacity", 1)
+                .style("z-index", 1);
```

Comment:
Added "z-index" to fix issue moseover with the tooltip (tooltip was hidding the buubles).